tests/client/client_test.go

```diff
@@ -49,29 +60,47 @@ func TestEncryptDecrypt(t *testing.T) {
 		{"GUID", []byte("b32a58c6-48c1-4552-8ff0-47680f3416d0"), []byte("b32a58c6-48c1-4552-8ff0-47680f3416d0")},
 	}
 
-	teardownTestCase := setupTestCase(t)
-	defer teardownTestCase(t)
-
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			encryptRequest := kmsv1.EncryptRequest{Version: version, Plain: tc.want}
-			encryptResponse, err := client.Encrypt(context.Background(), &encryptRequest)
+			v1EncryptRequest := kmsv1.EncryptRequest{Version: version, Plain: tc.want}
+			v1EncryptResponse, err := v1Client.Encrypt(context.Background(), &v1EncryptRequest)
```

Comment:
while we're here, could you define a context with timeout and use that instead of `context.Background()`?